examples/merkle-tree-cli/sample_tree.csv

```diff
@@ -0,0 +1,3 @@
+0x12345
+0x6789A
+0xBCDEF
```

Comment:
I don't really know if this changes anything, but this is not a `csv` file.

Reply:
While it could be argued that it's a [valid csv](https://en.wikipedia.org/wiki/Comma-separated_values#:~:text=CSV%20is%20a%20delimited%20data%20format%20that%20has%20fields/columns%20separated%20by%20the%20comma%20character%20and%20records/rows%20terminated%20by%20newlines.) considering that each element is read as a row and not a column, I understand that a `csv` file that separetes elements by commas is the more standarized approach. I have made the changes necessary to make it work that way.

Reply:
thanks!